### ISSUE TITLE
YAMLParser does not handle list-like YAML texts

### ISSUE THREAD

**Issue by fcami** [RedHatInsights/insights-core#2060]
If a list-like YAML text is parsed a backtrace occurs:
```
insights/core/__init__.py:87: in __init__
    self._handle_content(context)
insights/core/__init__.py:90: in _handle_content
    self.parse_content(context.content)
insights/core/__init__.py:700: in parse_content
    self.doc = from_dict(self.data)
insights/parsr/query/__init__.py:476: in from_dict
    return Entry(children=inner(orig))
(...)
>       for k, v in d.items():
E       AttributeError: 'list' object has no attribute 'items'
```

**Comment by miclark** [comment]
Assigning to @fcami since he seems to be working the issue.

**Comment by bfahr** [comment]
Fixed by #2061
